capture_service/android_application.h

```diff
@@ -51,6 +51,15 @@ class AndroidApplication
     bool                 IsDebuggable() const { return m_is_debuggable; }
     bool                 IsStarted() const { return m_started; }
     virtual bool         IsRunning() const;
+    void                 gfxrEnabled(bool enable);
```

Comment:
it would be better to be consistent with existing naming conventions for function names (start with an upper case letter)